bahmni-commons-api/src/test/java/org/bahmni/module/bahmnicommons/api/dao/impl/PatientDaoImplLuceneIT.java

```diff
@@ -306,4 +306,81 @@ public void shouldSearchByAnyPatientFullName() {
         assertEquals("{\"National ID\" : \"NAT100010\"}", patient.getExtraIdentifiers());
     }
 
+    @Test
+    public void shouldSearchAnyPatientWithSingleCharacterFirstName() {
```

Comment:
What mode is this LUCENE test running in? STARTS WITH?

Reply:
I haven't added that configuration yet. I'll create a separate PR for it. Currently, for single characters, it's EXACT and ANYWHERE, and for multi-character, it's ANYWHERE.